src/constants-nls.ts

```diff
@@ -0,0 +1,25 @@
+import { localize } from "./localize";
+
+export const viewOutput: string = localize('viewOutput', 'View Output');
+export const previewDescription: string = localize('preview', '(Preview)');
+export const pythonNewModelPreview: string = localize('pythonNewModelPreview', 'Python (Programming Model V2)');
+export const hostJsonConfigFailed: string = localize('durableStorageConfigFailed', 'WARNING: Failed to configure your JSON files for durable storage, you may need to configure them manually or start from a clean project.');
+export const emptyWorkspace: string = localize('emptyWorkspace', 'Your workspace folder looks empty, please navigate to the root directory of your project.');
+export const useEmulator: string = localize('useEmulator', 'Use Local Emulator');
+export const skipForNow: string = localize('skipForNow', 'Skip For Now');
+export const overwriteRemoteConnection = (a: string) => localize('overwriteRemoteConnection', 'We detected a different local connection setting for "{0}" than what was previously used. Would you like to overwrite your remote setting?', a);
+
+export const invalidAlphanumericWithHyphens: string = localize('invalidAlphanumericOrHyphen', `A name must consist of alphanumeric characters or '-', and must start and end with an alphanumeric character.`);
```

Comment:
Just an FYI, but both of these "invalid" messages only are used in one place each currently; however, I put the constants here because I wanted them to be easily discoverable by devs since they are meant for reuse and match up with the common `validateUtils` methods I added.